Jobsway2goMvc/Views/Groups/Details.cshtml

```diff
@@ -67,3 +79,93 @@
     <a asp-action="Edit" asp-route-id="@Model?.Id">Edit</a> |
     <a asp-action="Index">Back to List</a>
 </div>
+
+@section Scripts
+{
+    <style type="text/css">
+    .switch {
+      position: relative;
+      display: inline-block;
+      width: 60px;
+      height: 34px;
+    }
+    
+    .switch input {display:none;}
+    
+    .slider {
+      position: absolute;
+      cursor: pointer;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-color: #ccc;
+      transition: .4s;
+    }
+    
+    .slider:before {
+      position: absolute;
+      content: "";
+      height: 26px;
+      width: 26px;
+      left: 4px;
+      bottom: 4px;
+      background-color: white;
+      transition: .4s;
+    }
+    
+    /* include generated hidden field here */
+    input[type="checkbox"]:checked + input[type="hidden"] + .slider,
+    input[type="checkbox"]:checked + .slider {
+      background-color: #2196F3;
+    }
+    
+    /* include generated hidden field here */
+    input[type="checkbox"]:focus + input[type="hidden"] + .slider,
+    input[type="checkbox"]:focus + .slider {
+      box-shadow: 0 0 1px #2196F3;
+    }
+    
+    /* include generated hidden field here */
+    input[type="checkbox"]:checked + input[type="hidden"] + .slider:before,
+    input[type="checkbox"]:checked + .slider:before {
+      transform: translateX(26px);
+    }
+    
+    /* Rounded sliders */
+    .slider.round {
+      border-radius: 34px;
+    }
+    
+    .slider.round:before {
+      border-radius: 50%;
+    }
+    </style>
+
+    <script type="text/javascript">  
+    $(function () {  
+      
+        $("#IsPublic").change(function () {  
+            
+             var Id = $(".switch-handle").data('id');
+             var isPrivate = $(this).is(':checked');
+             
+             var url = '@Url.Action("UpdatePrivacy", "Groups")/' + Id;
+            $.ajax({  
+                type: "POST",  
+                url: url, 
+                data: JSON.stringify(isPrivate),  
+                contentType: "application/json",  
+                success: function (response) {  
+                   //console.log('success ', response);
+                },  
+                error: function (error) {  
+                 //console.log('error ', error);
+                }  
+            });  
+ 
+            return false;  
+        });  
+    });   
+    </script>
```

Comment:
Create a new js file inside `wwwroot/js` folder and name it `group.js`. Put this code inside and look if it works as it should. If needed, wrap it in a `$(document).ready(function() { // code here })`

Reply:
did these changes, thanks